assets/js/util/date-range.js

```diff
@@ -15,35 +15,36 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 /**
  * WordPress dependencies
  */
-import { applyFilters } from '@wordpress/hooks';
 import { __, _n, sprintf } from '@wordpress/i18n';
 
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+
 /**
  * Gets the current dateRange string.
  *
+ * @param {string} [slug] Optinal. The date range slug.
  * @return {string} the date range string.
  */
-export function getCurrentDateRange() {
-	/**
-	 * Filter the date range used for queries.
-	 *
-	 * @param String The selected date range. Default 'Last 28 days'.
-	 */
-	const dateRange = applyFilters( 'googlesitekit.dateRange', 'last-28-days' );
+export function getCurrentDateRange( slug ) {
+	const dateRange = slug || getCurrentDateRangeSlug();
```

Comment:
Since we don't need both `slug` and `dateRange` let's use it as the default value instead 👍 
```suggestion
export function getCurrentDateRange( dateRange = getCurrentDateRangeSlug() ) {
```
Let's update the doc block to refer to this as `dateRange` too for consistency with how it is called.

Reply:
Updated.